src/NBitcoin/Payment/BitcoinUrlBuilder.cs

```diff
@@ -230,7 +230,7 @@ private static void WriteParameters(Dictionary<string, string> parameters, Strin
                     builder.Append("&");
                 builder.Append(parameter.Key);
                 builder.Append("=");
-                builder.Append(HttpUtility.UrlEncode(parameter.Value));
+                builder.Append(System.Web.NBitcoin.HttpUtility.UrlEncode(parameter.Value));
```

Comment:
This namespace seems weird? Does ```System.Web.NBitcoin``` exist?

Reply:
Indeed it does... https://github.com/stratisproject/StratisBitcoinFullNode/blob/master/src/NBitcoin/Payment/HttpUtility.cs We might be able to replace it with System.Web.HttpUtility Will give it a try. I did some google searching and there seemed to be some drama in the past with accessing it in certain version of .net framework which doesn't appear to be a problem anymore. So I suspect that code was to work around it. I will run some tests and if it hangs together get rid of the offending code and update this PR.

Reply:
I think best to leave as is. Changing it to use System.Web.HttpUtility then breaks CanParsePaymentUrl() test. It seems System.Web.HttpUtility escapes symbols slightly different which is probably harmless but since I know nothing about what actually consumes this functionality it's probably best to leave it as is so the tests pass.

Reply:
there, i fixed it #1003